stdlib/hashlib.pyi

```diff
@@ -1,17 +1,19 @@
 import sys
-from _typeshed import ReadableBuffer
+from _typeshed import ReadableBuffer, Self
 from typing import AbstractSet
 
 class _Hash(object):
-    digest_size: int
-    block_size: int
-
+    @property
+    def digest_size(self) -> int: ...
+    @property
+    def block_size(self) -> int: ...
     # [Python documentation note] Changed in version 3.4: The name attribute has
```

Comment:
This comment can be removed, we don't care about Python < 3.4 anyway.

Reply:
Done as of https://github.com/python/typeshed/pull/6031/commits/e4291458236d779e95291b0e1643fec53790fa19.